engine/order_manager_types.go

```diff
@@ -37,11 +41,12 @@ type orderManagerConfig struct {
 
 // store holds all orders by exchange
 type store struct {
-	m               sync.RWMutex
-	Orders          map[string][]*order.Detail
-	commsManager    iCommsManager
-	exchangeManager iExchangeManager
-	wg              *sync.WaitGroup
+	m                         sync.RWMutex
```

Comment:
I've added this ez trello ticket [here](https://trello.com/c/8G54M6Ei/141-investigate-and-change-order-manager-mutex-from-rwmutex-to-mutex). I think it could derail things in this PR, but its a good point you brought up

Reply:
Thank you sir!